### DISCUSSION TITLE
Nucoin

### DISCUSSION THREAD

**Discussion by Kiny-Kiny** [andreroggeri/pynubank#417] [Ideas]
Vocês pretendem fazer implementação de alguma função para mostrar ao vivo o gráfico da Nucoin e também fazer uma implementação de compra/venda do mesmo?

**Comment by andreroggeri** [comment]
Eu não tenho a intenção de implementar o gráfico (Acredito que seria mais fácil obter essa informação de alguma fonte de preço de crypto). Mas PRs são bem vindos

Sobre a compra/venda seria necessário implementar a segunda autenticação (Envio da selfie) e não acredito que seja simples.

**Reply by Kiny-Kiny** [reply]
Certo. É que estou vendo muitos fazendo engenharia reversa dentro do App para obter o gráfico, pois oficialmente só há o gráfico no próprio App e ele não está presente em nenhuma outra plataforma.

**Reply by andreroggeri** [reply]
Imagino que seja alguma query graphql ou algum request especifico que é exposto no discovery.
Se alguém quiser abrir um PR com a implementação aqui, acredito que seja simples.
